### PR TITLE
UTY-2657: set max instances to 12 for windwos, linux and mac

### PR DESCRIPTION
**Pull request by BryanJY-Wong**
#### Description
set max ci agents to 12

#### Tests
How did you test these changes prior to submitting this pull request?
What automated tests are included in this PR?

#### Documentation
How is this documented (for example: release note, upgrade guide, feature page, in-code documentation)?

**Did you remember a changelog entry?**

#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

File: .buildkite/premerge.steps.yaml [hunk 1]
```diff
@@ -19,6 +19,7 @@ windows: &windows
     - "agent_count=1"
     - "machine_type=quad"
     - "experiment_normalised_upload_paths=true"
+    - "maximum_instances=12"
```
Comment (paulbalaji) [inline .buildkite/premerge.steps.yaml:22]: @zeroZshadow do we need the `"health_check_enabled=false"` here?

Comment (jamiebrynes7) [reply .buildkite/premerge.steps.yaml:22]: I believe we should yeah

Comment (zeroZshadow) [reply .buildkite/premerge.steps.yaml:22]: Yes Though I thought @seanjparker had already done this?

Comment (paulbalaji) [reply .buildkite/premerge.steps.yaml:22]: Looks like he hit core/fps but missed the blank project

Comment (paulbalaji) [reply .buildkite/premerge.steps.yaml:22]: yeah forgot the health_check one here https://github.com/spatialos/gdk-for-unity-blank-project/pull/109/files

File: .buildkite/release-qa.steps.yaml [hunk 2]
```diff
@@ -19,6 +19,7 @@ windows: &windows
     - "agent_count=1"
     - "machine_type=quad"
     - "experiment_normalised_upload_paths=true"
+    - "maximum_instances=12"
```
Comment (jamiebrynes7) [inline .buildkite/release-qa.steps.yaml:22]:
```suggestion
    - "health_check_enabled=false"
    - "maximum_instances=12"
```

**Pull request merged**